netcore/Microsoft.Azure.Storage.DataMovement/Microsoft.Azure.Storage.DataMovement.csproj

```diff
@@ -11,7 +11,7 @@
     <AssemblyName>Microsoft.Azure.Storage.DataMovement</AssemblyName>
     <AssemblyOriginatorKeyFile>../../tools/strongnamekeys/fake/windows.snk</AssemblyOriginatorKeyFile>
     <SignAssembly>true</SignAssembly>
-    <PackageId>Microsoft.WindowsAzure.Storage.DataMovement</PackageId>
+    <PackageId>Microsoft.Azure.Storage.DataMovement</PackageId>
```

Comment:
Are we changing the package ID on purpose?

Reply:
Yes. This is just clean up really. I was going through this file and saw that this never got changed back. This was the previous name when the package was still in beta.

This doesn't affect any releases. I just changed it for consistency.